apps/cow-fi/components/Chart/LoadingChart.tsx

```
@@ -1,5 +1,5 @@
 import styled from 'styled-components/macro'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)